client/src/components/Form/Form.js

```diff
@@ -1,38 +1,52 @@
-import React, { Component } from 'react';
+import React, { useEffect, useRef } from 'react';
 import FormAlert from 'components/FormAlert/FormAlert';
 import PropTypes from 'prop-types';
 
-class Form extends Component {
-  constructor(props) {
-    super(props);
-    this.handleSubmit = this.handleSubmit.bind(this);
-  }
+const Form = ({
+  autoFocus,
+  setDOM = () => null,
+  valid,
+  actions,
+  fieldHolder,
+  actionHolder = {
+    className: 'btn-toolbar'
+  },
+  afterMessages,
+  attributes,
+  fields,
+  handleSubmit,
+  mapActionsToComponents,
+  mapFieldsToComponents,
+  messages,
+  formTag: FormTag = 'form',
```

Comment:
Aliased as FormTag when destructured - previously was `const FormTag = this.props.formTag;`